<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivateChatWid</class>
 <widget class="QWidget" name="PrivateChatWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="priChatName_l">
       <property name="font">
        <font>
         <family>Adobe Devanagari</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>私聊对象</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="showMsg_te">
       <property name="font">
        <font>
         <family>Adobe Devanagari</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="inputMsg_le">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendMsg_pb">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
